app/src/core/profile/ProfilesController.ts

```diff
@@ -22,28 +23,27 @@ export default class ProfilesController extends EventEmitter {
 
     // Получение объекта пользователя
     public getProfile(userId: string = MY_ID): Profile {
-        if (!this._profiles.has(userId)) {
-            throw new Error(`Профиля с id: ${userId} не существует.`);
+        if (this._profiles.has(userId)) {
+            return this._profiles.get(userId)!;
         }
 
-        return this._profiles.get(userId) as Profile;
+        this.emit(MainClientEvents.ERROR, `Профиля с id: ${userId} не существует.`);
+        return new EmptyProfile("", this._request, this._dispatch);
     }
 
     // Добавление нового профиля пользователя
-    public addProfile(userId: string = MY_ID): void {
```

Comment:
:void means it does not return somethin right? why we remove it?

Reply:
yes, it does not return something
i remove it, because it is not necessery (ts style guide)
if the returning type of function is simple - we don`t have to write it. It increases the code base